configs/distill/mmcls/byot/byot_logits_resnet18_cifar100_8xb16_in1k.py

```diff
@@ -0,0 +1,155 @@
+_base_ = [
+    '../../../_base_/datasets/mmcls/cifar100_bs16_auto_aug.py',
+    'mmcls::_base_/schedules/cifar10_bs128.py',
+    'mmcls::_base_/default_runtime.py'
+]
+
+optim_wrapper = dict(
+    optimizer=dict(type='SGD', lr=0.1, momentum=0.9, weight_decay=0.0005))
+param_scheduler = dict(
+    type='MultiStepLR', by_epoch=True, milestones=[80, 160, 240], gamma=0.1)
+train_cfg = dict(by_epoch=True, max_epochs=250, val_interval=1)
+
+model = dict(
+    _scope_='mmrazor',
+    type='SelfDistill',
+    data_preprocessor=dict(
+        type='ImgDataPreprocessor',
+        # RGB format normalization parameters
+        mean=[129.304, 124.070, 112.434],
+        std=[68.170, 65.392, 70.418],
+        # convert image from BGR to RGB
+        bgr_to_rgb=False),
+    architecture=dict(
+        type='mmcls.ImageClassifier',
+        backbone=dict(
+            type='mmcls.ResNet_CIFAR',
+            depth=18,
+            num_stages=4,
+            out_indices=(3, ),
+            style='pytorch'),
+        neck=dict(type='mmcls.GlobalAveragePooling'),
+        head=dict(
+            type='mmcls.LinearClsHead',
+            num_classes=100,
+            in_channels=512,
+            loss=dict(type='mmcls.CrossEntropyLoss', loss_weight=1.0))),
```

Comment:
Could we directly use `architecture=dict(cfg_path='mmcls::resnet/resnet18_8xb16_cifar10.py', pretrained=False)`

Reply:
MMCLS doesn't provide a `resnet18_cifar100` in `mmcls::resnet`. The model config dict is not loaded until `build_MODEL`, which means I cannot modify `architecture.head.num_classes` in config.py as in `mmcls:resnet.resnet50_8xb16_cifar100.py`.
An alternative option is to add a local config `mmrazor/configs/_base_/models/mmcls/resnet/resnet18_8xb16_cifar100.py`, but I'm not sure whether this is allowed.